src/algolia-projects.json

```diff
@@ -1,4 +1,16 @@
 [{
+    "name": "Atom.io",
```

Comment:
A guy suggested that you change the name to something more obvious because similar projects seem to have been discontinued. Could worth to check with the owners.
https://github.com/mehcode/awesome-atom/pull/54